scala/scala-impl/src/org/jetbrains/plugins/scala/codeInspection/scalastyle/ScalastyleCodeInspection.scala

```diff
@@ -89,6 +87,9 @@ object ScalastyleCodeInspection {
     private val maybeDocument = PsiDocumentManager.getInstance(file.getProject).getDocument(file).toOption
 
     def toProblemDescriptor(msg: Message[SourceSpec]): Option[ProblemDescriptor] = msg match {
+      case StyleException(_, _, message, _, None, None) =>
+        Some(manager.createProblemDescriptor(file, message, true, Array.empty[LocalQuickFix], ProblemHighlightType.GENERIC_ERROR))
```

Comment:
`GENERIC_ERROR` might be harsh since there are probably folks who don't run `scalastyle` as a blocker to code entry / passing tests.